test/bdd/fixtures/demo/docker-compose-third-party.yml

```diff
@@ -140,6 +140,7 @@ services:
           - org3.trustbloc.local
           - auth-rest.trustbloc.local
           - auth-rest-hydra.trustbloc.local
+          - authz-kms.trustbloc.local
```

Comment:
need to add this to the hosts setup script

Reply:
Its not required as its not called from the local machine. Call to authz-kms is originating inside the docker container.